internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/endToEnd/TestBatch.java

```diff
@@ -471,7 +471,7 @@ public void testIncrementalPushWithCompression(
     }, storeName, null);
   }
 
-  @Test(timeOut = TEST_TIMEOUT)
+  @Test(timeOut = TEST_TIMEOUT, enabled = false)
```

Comment:
Initially I saw this test failed during a hang run, but it turns out that it might just be flaky. Reverted. TestBatch has 1-2 tests flaky, but we can check back later as the top priority is to fix the build.